ui/qt-lib/resources/stylesheet.qss

```diff
@@ -316,6 +316,113 @@ QDialog QPushButton[styleRole="passwordVisibilityToggle"]:checked {
     background: #fafafa;
 }
 
+#settingsDialog QComboBox {
+    border: none;
+    background: transparent;
+    padding-right: 5px;
+    color: rgb(52, 52, 62, 127);
+}
+
+#settingsDialog QComboBox::drop-down {
+    border: none;
+    background: transparent;
+  }
+
+#settingsDialog QComboBox::down-arrow {
+    image: url(':/images/expand-arrow.png');
+}
+
+#settingsDialog QComboBox QAbstractItemView {
+    border: 1px solid rgba(0, 0, 0, 30);
+    background: #fafafa;
+}
+
+QPushButton[styleRole="settingsTopbarButton"] {
+    width: 120px;
+    height: 30px;
+    border: none;
+    background: transparent;
+    color: rgb(52, 52, 62, 59);
+}
+
+QPushButton[styleRole="settingsTopbarButton"]:hover {
+    background: rgba(52, 52, 62, 10);
+}
+
+QPushButton[styleRole="settingsTopbarButton"]:checked {
+    border: none;
+    background: rgba(0, 202, 157, 25);
+    color: #00ca9d;
+}
+
+QPushButton[styleRole="rightPaneSettingsButton"] {
+    border: none;
+    background: rgba(0, 202, 157, 25);
+    color: #00ca9d;
+}
+
+QPushButton[styleRole="rightPaneSettingsButton"]:hover {
+    border: none;
+    background: rgba(0, 202, 157, 60);
+    color: #00ca9d;
+}
+
+QPushButton[styleRole="rightPaneSettingsButton"]:pressed {
+    background: rgba(0, 202, 157, 255);
+    color: #ffffff;
+}
+
+QPushButton[styleRole="rightPaneSettingsButton"]:disabled {
+    background: rgba(0, 202, 157, 10);
+    color: rgba(0, 202, 157, 127);
+}
+
+QPushButton[styleRole="resetSettingsButton"] {
```

Comment:
I suppose we should introduce some generic names for this button styles. How about "inverseButton" and "inverseSecondaryButton". You can suggest better ones though :)